### PR TITLE
(maint) update some readmes

### PR DESCRIPTION
**Pull request by jtappa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,32 +41,29 @@ in.
...
 
-Support
--------
+## Support
 
 Please log tickets and issues at our [JIRA tracker](https://tickets.puppetlabs.com).  A [mailing
 list](https://groups.google.com/forum/?fromgroups#!forum/puppet-users) is
-available for asking questions and getting help from others. In addition, there
-is an active #puppet channel on Freenode.
+available for asking questions and getting help from others.
```
Comment (melissa) [inline README.md]: Should we include a referenced to slack here since we've replaced irc with that?

Comment (jtappa) [reply README.md]: @melissa updated!

File: README.md [hunk 2]
```diff
@@ -45,32 +41,29 @@ in.
...
-Support
--------
+## Support
 
-Please log tickets and issues at our [JIRA tracker](https://tickets.puppetlabs.com).  A [mailing
+Please log tickets and issues at our [JIRA tracker](https://tickets.puppetlabs.com). A [mailing
 list](https://groups.google.com/forum/?fromgroups#!forum/puppet-users) is
-available for asking questions and getting help from others. In addition, there
-is an active #puppet channel on Freenode.
+available for asking questions and getting help from others, or if you prefer chat, we also have a [Puppet Community slack.](https://puppetcommunity.slack.com/)
```
Comment (melissa) [inline README.md]: Specifically the #puppet-dev channel

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -159,9 +154,7 @@ ensure the issue has been resolved.
 
 * [Puppet community guidelines](https://puppet.com/community/community-guidelines)
 * [Bug tracker (Jira)](https://tickets.puppetlabs.com)
-* [Contributor License Agreement](http://links.puppet.com/cla)
+* [Contributor License Agreement](https://cla.puppet.com/)
 * [General GitHub documentation](https://help.github.com/)
 * [GitHub pull request documentation](https://help.github.com/articles/creating-a-pull-request/)
-* #puppet-dev IRC channel on freenode.org ([Archive](https://botbot.me/freenode/puppet-dev/))
```
Comment (melissa) [inline CONTRIBUTING.md]: Oh, we should also add a reference to slack here!

**Comment by melissa** [comment]
@jtappa I'm happy to merge this, but I do think it would be beneficial to reference the #puppet-dev slack channel as the primary way to ask questions and get help when contributing to puppet

**Comment by jtappa** [comment]
Ooops, sorry I thought I committed/pushed the update! 😬

**Pull request merged**